services/dataload/dataload.go

```diff
@@ -0,0 +1,45 @@
+package dataload
+
+import (
+	"academy/model"
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+)
+
+var allJokes []model.Joke
+
+//LoadData this will load the data from the csv
+func LoadData() {
+	allJokes = append(allJokes[:0])
+	pwd, _ := os.Getwd()
+	csvFile, err := os.Open(pwd + "/data/data.csv")
+	if err != nil {
+		fmt.Println(err)
+	}
+	defer csvFile.Close()
+	r := csv.NewReader(csvFile)
+	records, err := r.ReadAll()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
You would only like to do `os.Exit` on the main function, on other packages and layers, if something goes wrong then handle and return the error

Reply:
Moved the `os.Exit` to main.go